tmpl/index.js

```diff
@@ -1,5 +1,10 @@
 import { app, BrowserWindow } from 'electron';
 
+// Handle creating/removing shortcuts on Windows when installing/uninstalling.
+if (require('electron-squirrel-startup')) {
```

Comment:
Uhhh, turns out that the example code to run `electron-squirrel-startup` doesn't pass airbnb linting 🤷‍♂️ 

Should we just `eslint-ignore-line global-require`?

Reply:
Sounds good to me, would prefer this module exported a function instead of having a side effect require, but that's not really our problem 👍